### PR TITLE
docs(ic): updated business view for Industry Core

### PR DESCRIPTION
**Pull request by markuskeidl**

### CODE REVIEW

**Review by mhellmeier — CHANGES_REQUESTED** [review]
There needs to be a `.license` file for every image added to the KITs. See [TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08#self-created-images) for more information.

When also using our standard CC-BY-4.0 license, this would lead to put the image itself with all its content open source. Since there are external icons and company logos involved (like on the `DataChain_World.png`) I don't think this would be okay.

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 1]
```diff
@@ -1,84 +1,109 @@
...
+## Vision & Mission
+
+### Vision
+
+The goal of Catena-X is to empower data-driven use cases to address the current challenges of sustainability,
+ resiliency, and quality. In the automotive industry, these challenges predominantly revolve around optimizing physical parts, 
+ components, and materials. The Catena-X Industry Core describes how these parts and vehicles can be digitally represented and found in the Catena-X network.
+ It serves as the foundation to enable the seamless integration of multiple use cases. 
+ Reusing central components and standards (especially data provisioning of digital twins of parts) across different use cases promotes efficiency and interoperability. 
+ The Industry Core enables a data sovereign data exchange based on parts and vehicles and lays the foundation for compliance with supply chain regulations.
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:41]: data sovereign data exchange => sovereign data exchange

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 2]
```diff
@@ -1,84 +1,109 @@
...
+ Reusing central components and standards (especially data provisioning of digital twins of parts) across different use cases promotes efficiency and interoperability. 
+ The Industry Core enables a data sovereign data exchange based on parts and vehicles and lays the foundation for compliance with supply chain regulations. 
+ Each partner in each Industry Core compliant use case expands the Catena-X network and the realization of seamless data chains. 
+ As a result, cross-domain network effects and n-tier use cases become possible with each use case benefiting from all others.
+
+# ![DataChain](assets/DataChain_World.png)
+
+### Mission
+
+The Industry Core describes a component at type and/or instance level, makes it identifiable and discoverable in the network and enables traversing across several levels.
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:49]: describes a component => describes a physical part, component or material

Just to align with the wording above and to know what is meant with component.

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 3]
```diff
@@ -1,84 +1,109 @@
...
+
+# ![DataChain](assets/IndustryCore.png)
+
+In the current version, the KIT supports the creation of digital twins on a Type-level (part- / material-number, catalog part) and Instance-level (a specific vehicle, serialized part, batches etc.) as well as the logical linking to their sub-components (Bill of Material, BoM). The default visibility of digital twins and their respective semantic models follows the one-up/one-down principle. 
+
+Parts and vehicles on the Instance-level can be provided and linked to their subparts, when the part or vehicle has been built. This is why it is often also refered to the lifecycle phase BoM as built.
+Parts and vehicles at type level can be provided at the earliest when the part has been assigned to a supplier. The linking of subcomponents at this point in time is often referred to as BoM as planned. However, the digital representation of the part at type level can also be initiated at a later point in time and remains in place until the part is dismantled.
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:60]: Parts and vehicles at type level can be provided at the earliest when the part has been assigned to a supplier.

I do not understand this sentence. Why is it not possible to publish a part at type level once I have designed it and I am ready to produce and ship it to my clients?

Proposal:
Parts and vehicles at type level can be provided at the earliest when the part has been designed and is available for ordering as a catalog part.

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 4]
```diff
@@ -1,84 +1,109 @@
...
+# ![DataChain](assets/DataChain_OneUp_OneDown.png)
+
+The Industry Core KIT provides the necessary standards, aspect models, APIs, logics, and processes on how to implement the Industry Core. 
+All described specifications in the KIT are based on Catena-X standards like Data Space Connector, Asset Administration Shell (AAS), and dezentralised Digital Twin Registry (DTR). They refer to other Catena-X KITs like the Connector KIT (EDC), Data Chain KIT (Item Relation Ship, IRS) and Business Partner KIT to ensure interoperability and data sovereignty according to IDSA and Gaia-X principles.
+
+
+## Business Value & Benefits
+### Business Value
+
+The Industry Core enables a data sovereign data exchange based on parts and vehicles and the realization of n-tier data chains.
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:71]: data sovereign data exchange => sovereign data exchange

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 5]
```diff
@@ -1,84 +1,109 @@
...
+All described specifications in the KIT are based on Catena-X standards like Data Space Connector, Asset Administration Shell (AAS), and dezentralised Digital Twin Registry (DTR). They refer to other Catena-X KITs like the Connector KIT (EDC), Data Chain KIT (Item Relation Ship, IRS) and Business Partner KIT to ensure interoperability and data sovereignty according to IDSA and Gaia-X principles.
+
+
+## Business Value & Benefits
+### Business Value
+
+The Industry Core enables a data sovereign data exchange based on parts and vehicles and the realization of n-tier data chains. 
+It facilitates the implementation of part-related Catena-X use cases such as traceability, circular economy, PCF and many more. 
+
+By reusing the core- & enablement services and the uniform usage of digital twins
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:74]: By reusing => The reusing ... accelerates

File: docs-kits/kits/Industry Core Kit/page_business_view.mdx [hunk 6]
```diff
@@ -1,84 +1,109 @@
...
+
+By reusing the core- & enablement services and the uniform usage of digital twins 
+accelerates data providers and consumers to implement new use cases to the Catena-X ecosystem. 
+This leads to a much faster time-to-market and scalability of new value-driven use cases.
+
+### Todays Challenge
+Catena-X use cases and solutions are based on the provision and utilisation of data. To this end, the use cases often define their own infrastructure and semantics for data exchange. This leads to redundancies at data level, resulting in additional effort and complexity in data provision.
+
+The Catena-X network and the Industry Core create an industry-wide standard for data exchange by re-using central components and shared semantics.
+To do so, once is enough: Once EDC, BPN, and Decentralized Registry + digital twins according to standardized semantics are established, numerous use cases can be enabled by extending the tech-stack with domain specific data aspects / models.
```
Comment (DrDanielMetz) [inline docs-kits/kits/Industry Core Kit/page_business_view.mdx:82]: Change the symbol + => and

**Review by mhellmeier — APPROVED** [review]

**Pull request merged**